src/test/java/org/mockitousage/debugging/VerificationListenerCallBackTest.java

```diff
@@ -12,30 +11,22 @@
 import org.mockito.verification.VerificationEvent;
 import org.mockito.verification.VerificationMode;
 
-
 import java.lang.reflect.Method;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assert.assertNull;
 import static org.junit.Assert.fail;
 import static org.mockito.Mockito.*;
 
 public class VerificationListenerCallBackTest {
-    private Method invocationWanted;
-    private RememberingListener listener;
-    private MockitoFramework mockitoFramework;
-
-    @Before
-    public void setUp() throws NoSuchMethodException {
```

Comment:
@TimvdLippe so some tests use 0, 1 and 2 concrete listeners. This gets confusing and in the mock test the name of the listener needs to be mockXXX because listener is taken. I find the tests more easy to read and maintain when only common logic is in the setup. The most popular unit testing books would agree on this.
This way yes there is some duplication but very minimal. I thought about moving some other pieces to the setup but I felt this was more clean.
Personally I have been burned by this...
Does this make some sense to you?